hogli/cli.py

```diff
@@ -0,0 +1,269 @@
+"""hogli - Unified developer CLI for PostHog monorepo.
+
+All commands are defined in scripts_manifest.yaml and auto-discovered.
+Help output is dynamically generated from the manifest with category grouping.
+"""
+
+from __future__ import annotations
+
+from collections import defaultdict
+
+import click
+
+from hogli.commands import BinScriptCommand, CompositeCommand, DirectCommand
+from hogli.manifest import REPO_ROOT, load_manifest
+
+HEDGEHOG_ART = r"""
```

Comment:
nit, not an attack to your design skills: Not the biggest fan of this one, there's a cool one being used in the Array product, maybe we could standardize and use the same one?